addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5174,6 +5174,16 @@ registry.layout_column = SnippetOptionWidget.extend({
         // Add the new column and update the grid height.
         rowEl.appendChild(newColumnEl);
         gridUtils._resizeGrid(rowEl);
+
+        // Scroll to the new column if more than half of it is hidden (= out of
+        // the viewport or hidden by an other element).
+        const newColumnPosition = newColumnEl.getBoundingClientRect();
+        const middleX = (newColumnPosition.left + newColumnPosition.right) / 2;
+        const middleY = (newColumnPosition.top + newColumnPosition.bottom) / 2;
+        const sameCoordinatesEl = this.ownerDocument.elementFromPoint(middleX, middleY);
+        if (!sameCoordinatesEl || !newColumnEl.contains(sameCoordinatesEl)) {
```

Comment:
In which case there is no `sameCoordinatesEl` ? :thinking:

Reply:
If the new grid item is out of the viewport :slightly_smiling_face: It return null in that case

Reply:
Ok thanks !